src/pages/Home/HomeMainContent.jsx

```diff
@@ -302,6 +302,13 @@ export default function HomeMainContent({
       </Box>
     )
   }
+  
+  const getAreaNameById = (id) => {
+    const area = areas.find((area) => area.id === id)
+    return area.name
+  }
+  const areaName = containerSelected ? getAreaNameById(containerSelected.areaId) : '';
```

Comment:
hice todo esto pq el BE no nos manda el area name con cada contenedor. solo manda el areaId